utils/test-server.js

```diff
@@ -33,10 +33,12 @@ app.get('/test/iframe/:random', function(req, res) {
 });
 
 app.get('/__echo/filename/:file', function(req, res) {
+  res.setHeader('Cache-Control', 24 * 60 * 60 * 1000);
   res.send(req.params.file);
 });
 
 app.get('/__echo/date/:file', function(req, res) {
+  res.setHeader('Cache-Control', 24 * 60 * 60 * 1000);
   res.send(`${req.params.file}-${Date.now()}`);
 });
```

Comment:
On the whole the tests here looked fine to me. Very minor comment about console messaging and another on consistent use of equality statements.